### DISCUSSION TITLE
MC log period

### DISCUSSION THREAD

**Discussion by mbsouth** [Ylianst/MeshCentral#5778] [Q&A] [answered]
Is it possible to set the MC log period to 365 days? 
If so, which parameters are responsible for that?

&#128153;&#10084;MC

**Comment by si458** [comment, marked as answer]
try below (this will set the database to store events for 365 days, as the default is only 20 days,
also the web ui will only show a max of 1000 events anyways
```
{
  settings: {
    dbExpire: {
      "events": 31536000
    }
  }
}
```
![image](https://github.com/Ylianst/MeshCentral/assets/765314/0507c07d-195a-4dd3-adb0-0222c3953871)

**Reply by mbsouth** [reply]
Thanks, works as expected!

**Reply by si458** [reply]
Just remember the more log events u store the bigger ur database will become in size! Provision more storage before hand
